scripts/examples/nrfconnect_example.sh

```diff
@@ -16,12 +16,15 @@
 #    limitations under the License.
 #
 
-# Run bootstrap to set up e.g. Pigweed correctly
+# Run bootstrap and activate to set up e.g. Pigweed correctly
 source "$(dirname "$0")/../../scripts/bootstrap.sh"
+source "$(dirname "$0")/../../scripts/activate.sh"
```

Comment:
isn't running bootstrap.sh enough?

Reply:
I think not for pigweed-app, as activate.sh sets proper python version. Without running activate.sh python version is set on the wrong version (I guess it is 2.7 ?) and when compiling pigweed syntax errors appear.

Reply:
Well, that's weird given that activate.sh is a symlink to bootstrap.sh:

```$ ll ./scripts/activate.sh 
lrwxrwxrwx 1 szatmz primarygroup 12 Nov  2 17:48 ./scripts/activate.sh -> bootstrap.sh
```

Reply:
@kkasperczyk-no can you take a peek at this? Sounds like something isn't working as expected..